sources/models/README.md

```diff
@@ -24,63 +24,73 @@ The `#[model]` attribute on Settings and its sub-structs reduces duplication and
 
 ### aws-k8s-1.23: Kubernetes 1.23
 
-* [Model](src/aws-k8s-1.25/mod.rs)
-* [Default settings](src/aws-k8s-1.25/defaults.d/)
+* [Model](src/aws-k8s-1.29/mod.rs)
+* [Default settings](src/aws-k8s-1.29/defaults.d/)
```

Comment:
These aren't the correct model links for these variants.

Can you fix these to point to the same targets as the links in `sources/models/src`?

Reply:
Yeah, Something wrong on my vscode. I'll fix it.